src/BizHawk.Emulation.Cores/Consoles/Nintendo/QuickNES/QuickNES.cs

```diff
@@ -215,6 +248,12 @@ static uint PackFourscoreButtonsFor(int portNumber, IController controller)
 			}
 		}
 
+		// Controller types
+		// none - 0
+		// joypad - 1
+		// arkanoidNES - 2
+		// arkanoidFamicom - 3

```

Comment:
Why is this not an enum or at least `const`s? And can you not re-use the `Port1PeripheralOption` for this too?